modules/spotxBidAdapter.js

```diff
@@ -202,14 +202,16 @@ export const spec = {
         userExt.consent = bidderRequest.gdprConsent.consentString;
 
         if (typeof bidderRequest.gdprConsent.gdprApplies !== 'undefined') {
-          requestPayload.regs = {
-            ext: {
-              gdpr: (bidderRequest.gdprConsent.gdprApplies ? 1 : 0)
-            }
-          };
+          requestPayload.regs = requestPayload.regs || {ext: {}};
+          requestPayload.regs.ext.gdpr = (bidderRequest.gdprConsent.gdprApplies ? 1 : 0);
         }
       }
 
+      if (bidderRequest && bidderRequest.uspConsent) {
+        requestPayload.regs = requestPayload.regs || {ext: {}};
```

Comment:
This isn't needed.
`utils.deepSetValue` can be used to set `regs.ext.us_privacy` [docs](https://www.npmjs.com/package/dset)

Reply:
Update to use `utils.deepSetValue`